framework/di/Container.php

```diff
@@ -493,7 +495,7 @@ protected function mergeParams($class, $params)
      * Returns the dependencies of the specified class.
      * @param string $class class name, interface name or alias name
      * @return array the dependencies of the specified class.
-     * @throws InvalidConfigException if a dependency cannot be resolved or if a dependency cannot be fulfilled.
+     * @throws NotInstantiableException if a dependency cannot be resolved or if a dependency cannot be fulfilled.
```

Comment:
Why change the exception type? That is a break in BC, that serves no clear purpose to me.

Reply:
BC not break, because `NotInstantiableException` extends `InvalidConfigException`.

Reply:
Ah, my bad! In that case LGTM!